sdk/python/lib/pulumi/runtime/rpc.py

```diff
@@ -228,11 +234,12 @@ def do_resolve(value_fut: asyncio.Future, known_fut: asyncio.Future, value: Any,
             value_fut.set_result(value)
             known_fut.set_result(is_known)
 
+        # Important to note here is that the resolver's future is assigned to the resource object using the
+        # pre-translated name. When properties are returned from the engine, we must first translate the name
```

Comment:
Does "pre-translated name" mean "name before translation"?

Reply:
Yeah, it does.

Reply:
Can we change the language to "name before translation"?

Reply:
sure, will do.